libraries/AP_Notify/OreoLED_I2C.cpp

```diff
@@ -159,6 +163,20 @@ bool OreoLED_I2C::mode_failsafe_radio()
 }
 
 
+// Procedure for when Pixhawk is in GCS failsafe
```

Comment:
minor nitpick, not all the world is Pixhawk

Reply:
Removed all of those references from the file.